app/src/main/report/services/ReportsService.ts

```diff
@@ -22,3 +25,9 @@ export class ReportsService extends ReportLocalService {
         return this.post(`/reports/${name}/versions/${version}/publish/?value=false`);
     }
 }
+
+export enum ReportsCacheKeysEnum {
+    "reports" = "reports",
+    "versions" = "versions",
+    "versionDetails" = "versionDetails"
+}
```

Comment:
Nice, this feels much better and will be handy if we do want to invalidate cache at any of these levels :+1: